config/locales/en/providers.yml

```diff
@@ -182,6 +182,17 @@ en:
     other_assets:
       show:
         h1-heading: Does your client have any of the following?
+    outgoings_summary:
+      add_other_outgoings:
+        add_other_outgoings: Add another type of regular payment
+      index:
+        add_all_outgoings: Add payments you made from %{start_at} to %{finish_at}.
```

Comment:
Should this be 'payments your client made'?

Reply:
Yes. Good catch